Jenkinsfile.release

```diff
@@ -0,0 +1,85 @@
+#!groovy
+
+def retagImage(String image_url, String source_tag, String dest_tag) {
+    // Pulls down image with source_tag and re-tags with dest_tag
+    // before pushing back to repository
+
+    sh "docker pull ${image_url}:${source_tag}"
+    sh "docker tag  ${image_url}:${source_tag} ${image_url}:${dest_tag}"
+    sh "docker push ${image_url}:${dest_tag}"
+}
+
+node{
+   properties(
+       [parameters(
+           [string(defaultValue: 'develop', description: 'SHA of the commit you wish to deploy.', name: 'GIT_COMMIT', trim: false),
+            string(defaultValue: 'tnation-test', description: 'Tag of the Current release.', name: 'RELEASE_TAG', trim: false),
+            string(defaultValue: 'rasterfoundry-staging-docs-site-us-east-1', description: 'Location of swagger spec.', name: 'RF_DOCS_BUCKET', trim: false),
```

Comment:
Reminder to replace all references to `Staging|staging` with `Production|production`